### PR TITLE
Profiles and Whitelist for Hud Toggle

### PR DESCRIPTION
**Pull request by AnteaterGAF**
Bioshock, Alan Wake's American Nightmare, Dead Rising 2: OTR, Psychonauts, Fallout New Vegas, Gurumin: A Monstrous Adventure, Kung Fu Strike, Star Wars: Force Unleashed, The Last Tinker, Lara Croft Guardian of Light, Kane & Lynch 2, Lord of the Rings: War in the North, Outland, Overlord, Torchlight 2 Hud Toggle

### CODE REVIEW

**Comment by PeterTh** [comment]
Neat

**Comment by karasuhebi** [comment]
Wow, thanks for this!

File: pack/config/whitelist.txt [hunk 1]
```diff
@@ -76,6 +80,7 @@ ffxiv*                       || Final Fantasy XIV Launcher or Configurator
 FirefallClient               || Firefall
 FoxGame-Win32-Shipping       || Blacklight: Retribution
 fr-041_debris                || "Debris" by Farbrausch
+Game                         || Gurumin: A Monstrous Adventure
```
Comment (ebugusey) [inline pack/config/whitelist.txt]: Wouldn't this conflict with other games that have this exact exe name, like all RPGMaker games?

Comment (PeterTh) [inline pack/config/whitelist.txt]: It does. I really need to implement path-based detection at some point.
(Or devs need to give better names to their executables :P)

**Pull request merged**